pkg/resources/common/duration_test.go

```diff
@@ -63,12 +63,16 @@ func TestParseHumanDuration(t *testing.T) {
 
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
-			output, err := ParseHumanReadableDuration(tc.input)
+			output, err := ParseTimestampOrHumanReadableDuration(tc.input)
 			if tc.expectedErr {
 				assert.Error(t, err)
 			} else {
 				assert.Equal(t, tc.expected, output)
 			}
 		})
 	}
+
+	// Testing timestamp parsing separately
```

Comment:
Not really a problem, I guess, but why is this one done separately from the others?